### ISSUE TITLE
Pausing the scene in the editor (F7) has no effect

### ISSUE THREAD

**Issue by vybr** [godotengine/godot#14742]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
3.0 Beta 1 - Windows

**Issue description:**
Trying to pause the game in the editor, it doesn't pause at all even after multiple attempts.
This happens in 2.1.x also.

**Comment by Zireael07** [comment]
Confirming on beta 1, not sure about 2.1.4.

**Comment by carlosfvieira** [comment]
Tested in b08735f20987a6188fd54c5e2dcd072dc316b413

The shortcut works, but only if the editor has the focus... but yes, i think it should work when the play scene window has the focus also...

**Comment by vybr** [comment]
The button doesn't work at all. The shortcut should also work when the editor is not in focus, with a option to turn it off in case the game uses F7 for input.

**Comment by akien-mga** [comment]
We have now entered release freeze for Godot 3.0 and want to focus only on release critical issues for that milestone. Therefore, we're moving this issue to the 3.1 milestone, though a fix may be made available for a 3.0.x maintenance release after it has been tested in the master branch during 3.1 development. If you consider that this issue is critical enough to warrant blocking the 3.0 release until fixed, please comment so that we can assess it more in-depth.

**Comment by reduz** [comment]
F7 does a debugger break. If there is no code running, I think no pausing happens. WIll check, anyway.

**Comment by reduz** [comment]
Well, does not seem like a bug, F7 is a debugger break.. I am not sure if Godot even supports pausing whole execution, so this seems more like an ehnacement to me. One idea was allowing to pause the game to actually be able to explore it (copy the camera from the editor to the game, since game does not run in the editor unlike Unity). Will kick to 3.2 I guess.

**Comment by akien-mga** [comment]
I can't reproduce the issue, pressing the button or F7 in the editor does pause execution and show the debugger. So it behaves as expected IMO.

I'll improve the tooltip as suggested.

**Comment by elXill** [comment]
I have also encountered this issue in v4.4.1.stable.official [49a5bc7b6] 

The problem for me is when play window is the active F7 doesn't work.
But it works in other ways such as;

- The button itself work when clicking. 
- Clicking and making editor window active then pressing F7 also works.

My impression with how F8 works, it should also work when play window is active. Similarly F5, F6 doesn't work when the play window is active either. (Unless I have made some changes in the settings and not remember doing so)

Also I am using a seperate play project window incase it makes a difference
